### PR TITLE
Deploy ellandi standalone app with api and web apps

### PR DESCRIPTION
**Pull request by rickyp72**
This is an intermediate step before switching to the single standalone ellandi app. NEXT_PUBLIC_API_URL is being implicitly set in the deploy.yml for non-prod envs. This is not set for prod as we don't expect to run all apps in prod.

### CODE REVIEW

File: .envs/ellandi [hunk 1]
```diff
@@ -0,0 +1,17 @@
+DEBUG=True
+DJANGO_SETTINGS_MODULE=ellandi.settings
+DJANGO_SECRET_KEY="django-insecure-&3_9ebzl!$@-vv=v=u@6h=lm@abd69u&cfs4lj0ml0v$thw*f3"
+<<<<<<< HEAD
```
Comment (Singletoned) [inline .envs/ellandi:4]: There seems to be some merge conflicts in this branch...

**Comment by Singletoned** [comment]
There's lots of merge problems and duplicate commits in this branch.  We need to have a look at how you're doing rebases because something is going wrong.

**Comment by Singletoned** [comment]
I've tidied this up a bit to remove some of the duplicate commits

**Comment by rickyp72** [comment]
Thanks @Singletoned - Yes, I think it got a bit confused as I rebased off sandbox and then develop forgetting that develop was behind your original branch. I think that might have been the issue.

**Pull request closed without merging**